### ISSUE TITLE
Redis taking memory issues

### ISSUE THREAD

**Issue by tommorgan365** [OpenCTI-Platform/docker#78]
Hello,
Default  opencti docker 5.5.4 install.  Looks like redis is consuming most of the memory then crashing.  Is there a way to limit redis ?

**Comment by ckane** [comment]
Yes, there is a `redis.trimming` configurable in the `production.json`. If you're using the docker compose deployment, here's an example of how to set it there:
* https://github.com/ckane/opencti-docker/blob/tf-main/docker-compose.yml#L48

Since the 5.5.4 release, I believe there has been a default to set trimming to 2M entries, which is what I chose for the above, too.

**Comment by tommorgan365** [comment]
ckanes suggestion worked - closing
